util/EthHelper.js

```diff
@@ -257,64 +241,34 @@ class EthHelper {
       ]));
     }
     if (!t || !currentBlockNumber) throw new Error('Cannot find transaction');
-    if (t.value > 0) return this.getEthTransferInfo(txHash, t, currentBlockNumber);
-    if (t.to.toLowerCase() !== LIKE_COIN_ADDRESS.toLowerCase()) throw new Error('Not LikeCoin transaction');
     this.initAbiDecoder();
-    const decoded = abiDecoder.decodeMethod(t.input);
-    const isDelegated = decoded.name === 'transferDelegated';
-    const isLock = decoded.name === 'transferAndLock';
-    if (decoded.name !== 'transfer' && !isDelegated && !isLock) {
-      throw new Error('Not LikeCoin Store transaction');
-    }
-    const txTo = decoded.params.find(p => p.name === '_to').value;
-    let _to = web3Instance.utils.toChecksumAddress(txTo);
-    let _from = isDelegated ? decoded.params.find(p => p.name === '_from').value : t.from;
-    _from = web3Instance.utils.toChecksumAddress(_from);
-    let _value = decoded.params.find(p => p.name === '_value').value;
+    TxInfo.init(abiDecoder, web3Instance);
```

Comment:
Should not have to init a non-class, pass `abiDecoder, web3Instance` into each fn as param